### PR TITLE
added link to sample

### PR DESCRIPTION
**Pull request by Mikejo5000**

### CODE REVIEW

**Comment by nikhilkh** [comment]
:shipit:

**Comment by nikhilkh** [comment]
Sounds like there is a conflict with master - @Mikejo5001 Could you please rebase? Your diff should only contain the line you added.

**Comment by cordova-qa** [comment]
Cordova CI Build has completed successfully. 

**Commit**     - [Link](https://github.com/apache/cordova-plugin-file-transfer/pull/145/commits/fd8ced3c22264687c40fd6c839cbe9338008ae63)
**Dashboard** - [Link](http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1/)

| Builder Name  | Console Output | Test Report | Device Logs  |
|     :---:     |     :---:      |   :---:     |     :---:    |
| [Windows 8.1 Store]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-store/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-store/console) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-store/testReport/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-store/artifact/) |
| [Windows 10  Store]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-10-store/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-10-store/console) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-10-store/testReport/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-10-store/artifact/) |
| [Windows 8.1 Phone]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-phone/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-phone/console) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-phone/testReport/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=windows-slave,platformName=windows-8.1-phone/artifact/) |
| [iOS]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=ios/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=ios/console) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=ios/testReport/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=ios/artifact/) |
| [Android Mac]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=android/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=android/console) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=android/testReport/) | [Link]( http://cordova-ci.cloudapp.net:8080/job/cordova-plugin-file-transfer-pr/1//label=mac-slave,platformName=android/artifact/) |

**Comment by nikhilkh** [comment]
@daserge Can you please review and help merge?

File: README.md [hunk 1]
```diff
@@ -315,3 +319,253 @@ If you are upgrading to a new (1.0.0 or newer) version of File, and you have pre
...
+
+```js
+function upload(fileEntry) {
+    // !! Assumes variable fileURL contains a valid URL to a text file on the device,
+    var fileURL = fileEntry.toURL();
+
+    var success = function (r) {
+        console.log("Successful upload...");
+        console.log("Code = " + r.responseCode);
+        displayFileData(fileEntry.fullPath + " (content uploaded to server)");
```
Comment (daserge) [inline README.md]: I would define `displayFileData` or just replace it with `console.log`.

**Comment by daserge** [comment]
LGTM, minor comments

File: README.md [hunk 2]
```diff
@@ -315,3 +319,253 @@ If you are upgrading to a new (1.0.0 or newer) version of File, and you have pre
...
+        }
+    );
+}
+```
+
+If you just need to display the image, take the FileEntry to call its toURL() function.
+
+```js
+function displayImageByFileURL(fileEntry) {
+    var elem = document.getElementById('imageFile');
```
Comment (daserge) [inline README.md]: Does it make sense to name it like `imageElement` instead?

**Comment by nikhilkh** [comment]
@daserge Thanks for your review - can you make the improvements you suggest and help merge this in.

**Comment by daserge** [comment]
Merged in https://github.com/apache/cordova-plugin-file-transfer/commit/82c5af9b5dd6accbd6b2d25174d3c1f4e576e5b3
@Mikejo5000 could you please close the PR?

**Pull request closed without merging**